samples/java/appgengine-java11/simple-server/README.md

```diff
@@ -0,0 +1,70 @@
+# Using the Java Agent on a Simple One File Server App in Google App Engine Standard Java 11
+
+NOTE: This example was copied from
+[appengine-java11/custom-entrypoint](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/appengine-java11/custom-entrypoint)
+and modified for Snapshot Debugger Java agent use.
+
+
+## Setup
+See [Prerequisites](../README.md#Prerequisites).
+
+## Compile the Source With Full Debug Information Enabled
+
+```
+javac -source 11 -target 11 -g Main.java
+```
+
+## Install the Snapshot Debugger Java Agent
+
+```
+mkdir cdbg
+wget -qO- https://github.com/GoogleCloudPlatform/cloud-debug-java/releases/latest/download/cdbg_java_agent_gce.tar.gz | tar xvz -C cdbg
+```
+
+## Deploy to App Engine Standard
+
+Examine the app.yaml file, which specifies the -agentpath java option to load
+the agent. Deploy the app with the following:
+
+```
+gcloud app deploy
+```
+
+Make note of the following output entries, which should resemble the following:
+
+```
+[...snip]
+target service:              [default]
+target version:              [20221117t213436]
+target url:                  [https://<your-project-id>.appspot.com]
+[...snip]
+```
+
+The service and version will be used to identify your debuggee ID.
+
+## Navigate To Your App
+
+This will ensure the app is run, which will allow the agent to register itself
+with the Firebase backend. The URL should be provide in the `target url` output
+of the pprevious step.
+
+## Determine the Debuggee ID
```

Comment:
As long as we're going through the sample to this level, might be worth suggesting a snapshot filename & line number to see things in action.